README.rst

```diff
@@ -19,40 +43,14 @@ Ray comes with libraries that accelerate deep learning and reinforcement learnin
 .. _`Ray Tune`: http://ray.readthedocs.io/en/latest/tune.html
 .. _`Ray RLlib`: http://ray.readthedocs.io/en/latest/rllib.html
 
-
 Installation
 ------------
 
-- Ray can be installed on Linux and Mac with ``pip install ray``.
-- To build Ray from source, see the instructions for `Ubuntu`_ and `Mac`_.
-
-.. _`Ubuntu`: http://ray.readthedocs.io/en/latest/install-on-ubuntu.html
-.. _`Mac`: http://ray.readthedocs.io/en/latest/install-on-macosx.html
```

Comment:
Shouldn't we remove this documentation page?

Reply:
hm is it not removed?

Reply:
Oh my mistake.